### PR TITLE
Build several flavors of Fluence Node docker image

### PR DESCRIPTION
**Pull request by nahsi**

### CODE REVIEW

**Comment by nahsi** [comment]
```# fluence/node-distro on multiple-packages 
> docker images | grep fluence
fluence-bundle                                     latest                         1b1c31e082dd   11 minutes ago      732MB
fluence-ipfs                                       latest                         fe048cc44702   17 minutes ago      263MB
fluence                                            latest                         249b7f4c50bd   17 minutes ago      193MB```

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -5,113 +5,146 @@ on:
...
+
+            fluencelabs/fluence:minimal
+            fluencelabs/fluence:minimal_${{ env.RELEASE_VERSION }}
+            fluencelabs/fluence:minimal_${{ env.RELEASE_VERSION }}_${{ env.RUN_NUMBER }}
+
+            ## rich
+
+            fluencelabs/fluence:rich
+            fluencelabs/fluence:rich_${{ env.RELEASE_VERSION }}
+            fluencelabs/fluence:rich_${{ env.RELEASE_VERSION }}_${{ env.RUN_NUMBER }}
```
Comment (folex) [inline .github/workflows/release.yml:145]: let's add some description here maybe? Like
```
### ipfs

Has Fluence Rust Peer, and IPFS daemon as a sidecar
...

### minimal

Contains only Fluence Rust Peer (no aqua-ipfs, no ceramic adapter)

###

Contains Fluence Rust Peer, IPFS daemon as a sidecar, ceramic cli, bitcoin cli and geth cli

...
```

Also I'd add a table to README like
| Container flavor  | Fluence Rust Peer | IPFS daemon | aqua-ipfs | registry | binaries |
| ---------------   | ---------------   | ----------- |----------- |----------- |----------- |
| minimal           |        ✅         | ❌           | ❌        |  ✅       | curl     |
| ipfs              |        ✅         | ✅           | ✅        |  ✅       | curl, ipfs |
| rich              |        ✅         | ✅           | ✅        |  ✅       | curl, ipfs, ceramic, bitcoin cli, geth cli |

File: README.md [hunk 2]
```diff
@@ -4,13 +4,45 @@ The distributive and packaging of the Fluence node.
...
 
 ### Environmental variables
 
-| variable       | required | description            |
-| -------------- | -------- | ---------------------- |
-| `CERAMIC_HOST` |          | ceramic daemon address |
+| variable                                         | default           | description                                 |
+| ------------------------------------------------ | ----------------- | ------------------------------------------- |
+| `CERAMIC_HOST`                                   |                   | ceramic daemon address                      |
+| `IPFS_PATH`                                      | `/ipfs`           | IPFS node data directory                    |
```
Comment (folex) [inline README.md:41]:
```
# docker exec -it fluence-0 bash
root@3ac1cc1457fa:/# cd /ipfs
bash: cd: /ipfs: No such file or directory
```

I believe it is `/config/ipfs`, no?

File: s6/ipfs/etc/cont-init.d/20-ipfs [hunk 3]
```diff
@@ -12,20 +10,16 @@ if [ ! -d "/config/ipfs" ]; then
...
-mkdir -p \
-	/config/log/ipfs \
-	/run
+# create directories
+mkdir -p $IPFS_PATH $IPFS_LOG_DIR
 
-# permissions
-chown -R abc:abc /config
-chown -R abc:abc /.fluence
+chown -R abc:abc $IPFS_PATH $IPFS_LOG_DIR
```
Comment (folex) [inline s6/ipfs/etc/cont-init.d/20-ipfs:25]: what about `chown /.fluence`? not needed?

**Review by folex — APPROVED** [review]

File: s6/ipfs/etc/cont-init.d/20-ipfs (hunk 3 above)
Comment (nahsi) [reply s6/ipfs/etc/cont-init.d/20-ipfs:25]: It is in a separate file called `30-fluence`

File: .github/workflows/release.yml (hunk 1 above)
Comment (nahsi) [reply .github/workflows/release.yml:145]: Actually aqua-ipfs present even in the minimal container. Need to redo builtins installation script to be able to choose which services to install. Better do to it separately in another task.

**Pull request merged**